scripts/sha1sums-aarch64

```diff
@@ -1,6 +1,6 @@
 6fee67adbfed8db7a225be23ee9d90b5bd7f19e6  bionic-server-cloudimg-arm64.img
 786fe1c33588334e92b35c65e414da068df180bc  bionic-server-cloudimg-arm64.raw
-6e66f9f4b01adc72c884c1c1111e60afadc9c871  bionic-server-cloudimg-arm64.qcow2
+4318bdd54228299259640ba255423c5c4fb90425  bionic-server-cloudimg-arm64.qcow2
```

Comment:
I think the SHA shouldn't be changed.

Reply:
In docker of ubuntu20.04,qemu-img --version 
```
qemu-img version 4.2.1 (Debian 1:4.2-3ubuntu6.24)
Copyright (c) 2003-2019 Fabrice Bellard and the QEMU Project developers
```

In docker of ubuntu22.04,qemu-img --version
```
qemu-img version 6.2.0 (Debian 1:6.2+dfsg-2ubuntu6.8)
Copyright (c) 2003-2021 Fabrice Bellard and the QEMU Project developers
```
qemu-img convert -p -f raw -O qcow2 bionic-server-cloudimg-arm64.raw /root/workloads/bionic-server-cloudimg-arm64.qcow2
```
    (100.00/100%) 
```
sha1sum bionic-server-cloudimg-arm64.qcow2 
```
4318bdd54228299259640ba255423c5c4fb90425  bionic-server-cloudimg-arm64.qcow2
```